apps/openmw/mwsound/soundmanagerimp.hpp

```diff
@@ -114,9 +114,15 @@ namespace MWSound
         MWBase::SoundStreamPtr playVoice(DecoderPtr decoder, const osg::Vec3f &pos, bool playlocal);
 
         void streamMusicFull(const std::string& filename);
+        void advanceMusic(const std::string& filename);
+
         void updateSounds(float duration);
         void updateRegionSound(float duration);
         void updateWaterSound(float duration);
+        void updateMusic(float duration);
+
+        float mMusicFader;
```

Comment:
mMusicFader needs to be initialized in the constructor. Even if a variable isn't supposed to be used until it's set, it is easy to make a mistake and end up with uninitialised values somewhere. (Also, tools like coverity/cppcheck will always scream at you for members not initialized in constructor)

Actually, why is this even a member variable when it's only used as a constant?